index.js

```diff
@@ -0,0 +1,32 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+import express from "express";
+import morgan from "morgan";
+import cors from "cors";
+import { PrismaClient } from "@prisma/client";
+import { productRouter } from "./routes/product-route.js";
+import { articleRouter } from "./routes/article-route.js";
+import {
+  productCommentRouter,
+  articleCommentRouter,
+} from "./routes/comment-route.js";
+
+const prisma = new PrismaClient();
+
+const app = express();
+app.use(express.json());
+app.use(morgan("dev"));
+app.use(
+  cors({
+    origin: "*",
+  })
+);
+
+app.use("/products", productRouter);
+app.use("/articles", articleRouter);
+app.use("/products", productCommentRouter);
+app.use("/articles", articleCommentRouter);
```

Comment:
이러면 comment router가 제대로 라우팅 되지 않을 것 같습니다.
그리고 router 이름도 불분명 하고요. 굳이 나누자면 아래처럼 할 수 있을 것 같습니다.
```suggestion
app.use("/comment/sproducts", productCommentRouter);
app.use("/cooment/articles", articleCommentRouter);
```